docs/changelog.rst

```diff
@@ -3,7 +3,25 @@
 =========
 Changelog
 =========
-
+* :release:`1.13.0 <2019-02-13>`
+* :bug:`452` Restore prompts while retaining support for suppressing prompts .
+* :bug:`447` Pin requests-toolbelt to 0.8.0 to avoid attempting to use openssl
```

Comment:
Did we actually pin or did we denylist `0.9.0`? I thought denylisted it

Reply:
Yeah, see https://github.com/pypa/twine/blob/95ecde2e93792bdc01508fb049601cd8906d2069/setup.py#L76 We're only preventing us from installing 0.9.0 to be safe, not because it's a requirement

Reply:
I worked off the PR title, let me confirm.

Reply:
yep we explicitly avoid 0.9.0, will update.